packages/snap-preact-components/src/components/Atoms/Image/Image.stories.tsx

```diff
@@ -4,11 +4,152 @@ import { ArgsTable, PRIMARY_STORY } from '@storybook/addon-docs/blocks';
 
 import { Image, FALLBACK_IMAGE_URL, ImageProps } from './Image';
 import { componentArgs } from '../../../utilities';
-import { MockData } from '@searchspring/snap-shared';
-import { SearchResponseModel } from '@searchspring/snapi-types';
 
-const mockData = new MockData();
-let searchResponse: SearchResponseModel = mockData.search();
+const searchResponse = {
```

Comment:
Good catch here. Can we also include a comment here (and in FacetSlider.stories.tsx) to explain why we needed to hard-code the mock data? (ie. storybook unable to bundle `MockData` due to it using `fs`)